src/components/ProjectDetails.tsx

```diff
@@ -75,7 +76,7 @@ const ProjectDetails = ({ id }: Props) => {
           token && (
             <Grid item md={8}>
               <TokenView
-                tokenId={token.tokenId}
+                tokenId={token.id}
```

Comment:
don't think we need to change this and instead use the token.contract_address instead where we need the contract addy

Reply:
addressed with [7202d8b](https://github.com/ArtBlocks/artblocks-engine-react/pull/10/commits/7202d8bf8996b2b5c2c735f6bc4f98af38528f59)